src/components/Visit/readvisitinstance.tsx

```diff
@@ -22,18 +24,29 @@ const VisitInfo = ({
   notes = ''
 }: VisitInfoProps) => {
   return (
-    <div
-      className='justify-between overflow-hidden text-sm transition-all duration-300'
-      style={{ maxHeight: isOpen ? '100vh' : '0' }}
-    >
-      <p>Visit Type: {type}</p>
-      <p>Pet(s): {petNames}</p>
-      <p>Duration: {formatDuration(duration)}</p>
-      <p>Walk Distance: {walkDist.toFixed(3)} km</p>
-      <p>Commute Distance: {commuteDist.toFixed(1)} km</p>
-      <p>Commute Method: {commuteMethod}</p>
-      <p>Notes: {notes}</p>
-    </div>
+    <>
+      <div
+        className={`justify-between overflow-hidden text-sm transition-all duration-300 ${
+          isOpen ? 'max-h-screen' : 'max-h-0'
+        }`}
+      >
+        <p>Visit Type: {type}</p>
+        <p>Pet(s): {petNames}</p>
+        <p>Duration: {formatDuration(duration)}</p>
+        <p>Walk Distance: {walkDist.toFixed(3)} km</p>
+        <p>Commute Distance: {commuteDist.toFixed(1)} km</p>
+        <p>Commute Method: {commuteMethod}</p>
+        <p>Notes: {notes}</p>
+        <div className='my-2 flex justify-center gap-2'>
+          <Link href='/visit/incident'>
+            <Button size='medium'>Report incident</Button>
+          </Link>
+          <Link href='/visit/vet'>
```

Comment:
There's some bug here with using the `Button` and the `Link` components together. You may need to do something here with passing refs. Or put the link inside the button rather than the other way around.

![image](https://user-images.githubusercontent.com/34858205/211131962-83912bc0-390e-4bc5-a473-c3c1e8124ec3.png)